src/components/MessageList/Message/index.tsx

```
@@ -47,32 +44,56 @@
   const { id, displayName, text, uid, timestamp, type } = message;
 
   // Then conditionally access type-specific properties
-  let boardSize, gameBoardId, image;
+  let boardSize: number | undefined, gameBoardId: string | undefined, image: string | undefined;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)